EIPS/eip-1.mediawiki

```diff
@@ -86,45 +83,42 @@ Each EIP should have the following parts:
 
 * Implementations -- The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing code.
 
+* Simple Summary -- "If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.
+
 ==EIP Formats and Templates==
 
-EIPs should be written in mediawiki or markdown format. Image files should be included in a subdirectory for that EIP.
+EIPs should be written in [https://en.wikipedia.org/wiki/Help:Cheatsheet mediawiki] or [https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet markdown] format. Image files should be included in a subdirectory for that EIP.
```

Comment:
I am in favor of removing mediawiki support and having GitHub flavored markdown as the only format accepted. I have re-written EIP1 in markdown.